### ISSUE TITLE
Colons echoed multiple times by NVDA

### ISSUE THREAD

**Issue by krzysz00** [microsoft/terminal#12770]
### Windows Terminal version

1.13.10734.0

### Windows build number

10.0.22000.0

### Other Software

NVDA 2021.3.1 (eSpeak synthesizer, if relevant). In the settings, under Keyboard, "Speak typed characters" is on and "Speak typed words" is off. Under "Speech", "Punctuation/symbol level" is "most"

### Steps to reproduce

At a command prompt (I've tested this with Command Prompt, PowerShell, and Ubuntu), type a colon `:`

### Expected Behavior

NVDA should announce "colon"

### Actual Behavior

NVDA announces "colon" then, after a short delay, announces "colon" again, despite only one colon being input.

**Comment by zadjii-msft** [comment]
@carlos-zamora and @codeofdusk as FYIs

**Comment by codeofdusk** [comment]
Is this fixed in the NVDA 2022.1 beta?

**Comment by krzysz00** [comment]
@codeofdusk Looks like it. Or at least I couldn't reproduce the problem either with a portable copy of the beta or with latest master.

Got any guesses for what could be going on here?

**Comment by krzysz00** [comment]
Works fine on NVDA 2022, closing
